examples/gno.land/p/demo/uassert/uassert.gno

```diff
@@ -416,3 +419,317 @@ func NotEmpty(t TestingT, obj interface{}, msgs ...string) bool {
 	}
 	return true
 }
+
+// Nil asserts that the specified object is nil.
+func Nil(t TestingT, obj interface{}, msgs ...string) bool {
+	t.Helper()
+	if obj != nil {
+		return fail(t, msgs, "uassert.Nil: object should be nil")
+	}
+
+	return true
+}
+
+// NotNil asserts that the specified object is not nil.
+func NotNil(t TestingT, obj interface{}, msgs ...string) bool {
+	t.Helper()
+	if obj == nil {
+		return fail(t, msgs, "uassert.NotNil: object should not be nil")
+	}
+
+	return true
+}
+
+// Contains asserts that the specified string or list contains the specified substring or element.
+func Contains(t TestingT, container, element interface{}, msgs ...string) bool {
```

Comment:
Can you document that Contains will only work for `string` or `[]string`?

Reply:
I had made a minimalist implementation of this function, but I've modified it to handle more types.

https://github.com/gnolang/gno/pull/2547/commits/8e02d310e3f915b3273cdfa162acd7f0d6dc8a9a